glib-macros/src/lib.rs

```diff
@@ -17,3 +18,29 @@ pub fn genum_derive(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     let gen = genum_derive::impl_genum(&input);
     gen.into()
 }
+
+/// Derive macro for defining a [`BoxedType`]`::get_type` function and
+/// the [`glib::Value`] traits.
+///
+/// # Example
+///
+/// ```
+/// use ::glib_macros::GBoxed;
+/// use glib::prelude::*;
+/// use glib::subclass::prelude::*;
+/// use glib::{glib_boxed_type, glib_boxed_derive_traits};
```

Comment:
Needed so the doc test passes. What would be the good way to import the macros here as I cannot use `#[macro_use]
extern crate glib;`  

```
---- src/lib.rs - gboxed_derive (line 6) stdout ----
error[E0468]: an `extern crate` loading macros must be at the crate root
 --> src/lib.rs:11:1
  |
8 | extern crate glib;
  | ^^^^^^^^^^^^^^^^^^
```

Reply:
Why you don't like this use? It seems normal for edition2018.
But for better usability IMHO we need find way to glob-use all macros.

Reply:
Also `/// ##[macro_use] extern crate glib;` works fine for me, see https://doc.rust-lang.org/rustdoc/documentation-tests.html#documenting-macros

Reply:
Adding `glib_boxed_wrapper, glib_boxed_type, glib_boxed_derive_traits,` to glib::prelude make
`extern crate glib` and `use glib::{glib_boxed_type, ..}` unneeded.

Reply:
Indeed, I guess I didn't put the `extern crate` before the `use` when I tried yesterday.